src/modules/missionHelper/missionHelper.vue

```diff
@@ -882,34 +880,95 @@ export default Vue.extend<
                         })
                 );
             }
+
             if (missionSpecs?.additional) {
                 const optionalAlternatives = this.$m(
                     'vehicles.optional_alternatives'
                 ) as Record<string, Record<string, string>>;
                 Object.keys(optionalAlternatives).forEach(alt => {
-                    if (
-                        !optionalAlternatives[alt].not_customizable &&
-                        !this.settings.optionalAlternatives[alt]
-                    )
-                        return;
-                    if (
-                        missionSpecs?.additional.hasOwnProperty(alt) &&
-                        missionSpecs.additional[alt]
-                    ) {
-                        return Object.keys(optionalAlternatives[alt]).forEach(
-                            rep => {
+                    if (this.settings.optionalAlternatives[alt]) {
+                        if (
+                            missionSpecs?.additional.hasOwnProperty(alt) &&
+                            missionSpecs.additional[alt]
+                        ) {
+                            if (
+                                optionalAlternatives[alt].line_break &&
+                                missionSpecs?.requirements.hasOwnProperty(
+                                    optionalAlternatives[alt].based_on
+                                )
+                            ) {
+                                if (
```

Comment:
Maybe short this with a helper variable or https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator

Reply:
maybe you can, but I prefer the long form. But i changed it for you